lib/Agrammon/Model.pm6

```diff
@@ -379,4 +380,34 @@ class Agrammon::Model {
         });
         %$!distribution-map-cache
     }
+
+    # Create LaTeX docu
+    method create-docu( :%technical! ) {
+        my @sections;
+        for @!evaluation-order.reverse -> $module {
+
+            my @inputs;
+            for $module.input -> $input (:$name, :$description, *%) {
+                @inputs.push( %( :name(latex-escape($input.name)), :description(latex-escape($input.description)) ) );
+            }
+
+            my @outputs;
+            for $module.output -> $output (:$name, :$formula, :$description, *%) {
+                @outputs.push( %(:name(latex-escape($output.name)), :$formula, :description(latex-escape($output.description)) ) );
+            }
+
+            my $tax = latex-escape($module.taxonomy);
+
+            @sections.push( %(
+                :title("\\subsection\{$tax\}"), :description(latex-escape($module.description)),
```

Comment:
If you don't want to double backslashes and curlies everywhere but still want to interpolate with scalars, you can build a custom quote construct that only enables scalar variable interpolation: `Q:s"\subsection{$tax}"`

Reply:
Thanks, done.